### ISSUE TITLE
Prevent Flask from Logging Every Request Body to syslog

### ISSUE THREAD

**Issue by allella** [codeforgreenville/upstate_tech_cal_service#26]
/var/log/messages is filling up with huge API response bodies.  This is likely due to gunicorn inheriting and/or not filtering out Flask debugging level messages.

There's a Medium article that describes the problem, but that [article hard-codes the web server, gunicorn](https://medium.com/@trstringer/logging-flask-and-gunicorn-the-manageable-way-2e6f0b8beb2f), so there must be a better solution.

**Comment by allella** [comment]
I did a lot of work on this and the only thing I've figured out is it's not a Logging issue.

I can setup a custom log and make log errors emit, or not, based on the [log config / level set in a .ini file](https://docs.python-guide.org/writing/logging/).

What is happening is that [gunicorn](http://docs.gunicorn.org/en/latest/settings.html#errorlog) is directing all output (print, jsonify, or anything that hits stdout (standard output in Linux) to the Linux syslog (/var/log/messages).

I found countless issues on Gunicorn and Flask about how to direct debugging info to stdout or stderror or syslog, but zero about how to make Gunicorn not direct ALL responses to the syslog.

The issue could be with Flask, or Apache, or some combination of Flask + Gunicorn + Systemd + Apache handling of stdout and stderror.

It's not only annoying and bloating the logs, but is also a long-term security issue to log response objects in production.  We're dealing in public API info right now, but it's still a bad idea to let the logging of responses persist for all of these reasons.
